data/operators/power/line.json

```diff
@@ -7145,7 +7132,8 @@
       "matchNames": [
         "enso",
         "enso netz",
-        "sachsennetze"
+        "sachsennetze",
```

Comment:
Please check whether the matchName is correct, see https://www.wikidata.org/wiki/Q124370321

Reply:
SachsenNetze is a 100% owned subsidiary of SachsenEnergie (grid operation and electricity distribution have to be legally separate).

SachsenEnergie was previously placed here so I thought it would have been previously established that this should be used.